protos/processor.proto

```diff
@@ -103,6 +104,41 @@ message TpProcessResponse {
     bytes extended_data = 3;
 }
 
+// The request from the transaction processor to the validator append data
+// to a transaction receipt
+message TpAddReceiptDataRequest {
```

Comment:
All of these new protobuf messages should be added to `protos/state_context.proto`, not here. At some point, that file will need to be renamed `protos/execution_context.proto`.